lxd/events/connections.go

```diff
@@ -2,17 +2,7 @@ package events
 
 import (
 	"context"
-	"encoding/json"
```

Comment:
The commit message `lxd/events: Add EventListenerConnection interface ` is no longer relevant to the change (which is just removing code).